build/configs/rtl8721csm/tc/Make.defs

```diff
@@ -210,10 +211,10 @@ define MAKE_BOARD_SPECIFIC_BIN
   $(TOPDIR)/../build/configs/rtl8721csm/rtl8721csm_make_bin.sh
 endef
 
-KERNEL_BINARY_PATH='$(TOPDIR)/../build/output/bin/km0_km4_image2.bin'
 SECURE_HEADER_SIZE=32
 FLASH_BASE_ADDRESS=0x8000000
 FLASH_SIZE=8388608
+KERNEL_BINARY_PATH='$(TOPDIR)/../build/output/bin/${KERNEL}_${CONFIG_BOARD_BUILD_DATE}.bin'
```

Comment:
KERNEL_BINARY_PATH is name of binary here not path, so we can rename?

Reply:
It includes not only the binary name, but also the path.
We need the path also here, so I named like it, so it is better to keep this.
Thanks.

Reply:
How about moving this into `define MAKE_SAMSUNG_HEADER`?
Then this will be set after build.